js/tsmlfe.js

```diff
@@ -2,17 +2,21 @@
 	Javascript functions for Enhanced Feedback Single Meetings template
 */
 
-function switchVisible() {
-	if (document.getElementById('map')) {
+function switchVisible(approximate) {
+	var map_container = document.getElementById('map');
+	var requests_container = document.getElementById('requests');
 
-		if (document.getElementById('map').style.display == 'none') {
-			document.getElementById('map').style.display = 'block';
-			document.getElementById('requests').style.display = 'none';
+	if (map_container.style.display == 'none') {
+		if (approximate == 'no') {
+			map_container.style.display = 'block';
 		}
```

Comment:
think you want an `else` here and below. if `approximate == 'no'` you'll be setting display to `block` and immediately setting it back to `none`

Reply:
No I don't think that's correct. When toggling back and forth between the map and the requests displaying we want one div hidden when the other is showing. The twist to the simple toggling comes when we have an approximate address where we don't want to display the map at all.  So, we continue to display the map and hide the request div when there is no approximate address, but set both to hide when toggling off and conversely hide the map when it would otherwise show.

Reply:
yeah i think i misread it at first, disregard. i'll "resolve" it